dockerfile/Dockerfile

```diff
@@ -16,55 +16,60 @@
 #
 # ------------------------------------------------------------------------
 
+# set to latest Ubuntu LTS
 FROM ubuntu:16.04
 MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
-ENV DEBIAN_FRONTEND noninteractive
 
-# defined arguments and defaults
+# set user configurations
 ARG USER=wso2carbon
-ARG GROUP=wso2
+ARG USER_GROUP=wso2
+ARG USER_HOME=/home/${USER}
+# set dependant files directory
 ARG FILES=./files
-ARG USER_HOME=/home/wso2carbon
+# set jdk configurations
+ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+# set wso2 product configurations
 ARG WSO2_SERVER=wso2is
 ARG WSO2_SERVER_VERSION=5.3.0
 ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}.zip
-ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
-ARG JAVA_HOME=${USER_HOME}/java
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 
-RUN apt-get update \
-	&& apt-get install -y --no-install-recommends --no-install-suggests \
-       	zip \
-        unzip \
-        telnet \
-        curl \
-        iproute2 \
-	sudo && rm -rf /var/lib/apt/lists/*
+# install required packages
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends --no-install-suggests \
+    curl \
```

Comment:
Shall we get these packages into one line ?

Reply:
@DilanUA I think we should stick to having packages in multiple lines. This is primarily because this seems to be the practice adopted in Docker library Dockerfiles. Please see the following Dockerfile best practices guide example [1] and openJDK official Dockerfile [2], respectively.
[1]: [link](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#sort-multi-line-arguments)
[2]: [link](https://github.com/docker-library/openjdk/blob/a893fe3cd82757e7bccc0948c88bfee09bd916c3/8-jdk/Dockerfile#L19)
Plus, IMO this technique is better since no other package names are expected to be added.